src/lib/UserEventHelper.ts

```diff
@@ -49,6 +66,46 @@ export async function getEventData(eid: string) {
   return data;
 }
 
+export async function verifyUserInEvent(eid: string, uid: string) {
+  if (!eid || !uid) {
+    throw new Error('Invalid event id, or invalid uid provided.');
+  }
+
+  const snapshot = await getDb()
+    .collection(DB_PATHS.EVENT_USERS)
+    .where('eid', '==', eid)
+    .where('uid', '==', uid)
+    .get();
+
+  if (snapshot.docs.length !== 1) {
+    throw new Error('Ticket does not exist.');
+  }
```

Comment:
Lines `70` to `82` are the same as the `validateUserAndEvent()` function. Call that function instead and remove these lines. Keep everything below line `82`.